app/helpers/application_helper.rb

```diff
@@ -428,18 +433,17 @@ def users_role_change_description(version)
   end
 
   def subscription_change_description(version)
-    user = get_version_object(version).user
+    user = current_or_last_object_state(version.item_type, version.item_id).user
     user_name = user.name unless user.id.to_s == version.whodunnit
     version.event == 'create' ? "subscribed #{user_name} to" : "unsubscribed #{user_name} from"
   end
 
   def registration_change_description(version)
     if version.item_type == 'Registration'
-      user = get_version_object(version).user
+      user = current_or_last_object_state(version.item_type, version.item_id).user
     else
```

Comment:
It would be useful to have a comment on this line indicating which is the else scenario. I think right now it is only the EventsRegistration.